solr/core/src/java/org/apache/solr/security/PKIAuthenticationPlugin.java

```diff
@@ -75,6 +76,15 @@ public static void withServerIdentity(final boolean enabled) {
   }
 
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+  /**
+   * If a number has less than this number of digits, it'll not be considered a timestamp.
+   */
+  private static final int MIN_TIMESTAMP_DIGITS = 10; // a timestamp of 9999999999 is year 1970
```

Comment:
This range looks reasonable to parse a timestamp from now and years to come.

Considering it's expected to be a timestamp, we could have this be between 13 and 14 digits but guess that's just optimization that'd not really help with anything. Let's leave it at this :)

Reply:
Yes, I thought about this too. At the end, I decided to keep the limits pretty open to handle only the bad parsing edge case, and let the regular expiration code handle the case where there is in fact a number. I'm happy to change the limits if you think it makes sense.